smallConf.py

```diff
@@ -119,15 +119,27 @@
 sendLineUntilC = lambda shape : shape.nbRowsK # stop sending on a p2p link if at least this amount of samples passed
 perNeighborQueue = True # queue incoming messages to outgoing connections on arrival (as typical GossipSub impl)
 shuffleQueues = True # shuffle the order of picking from active queues of a sender node
-perNodeQueue = False # keep a global queue of incoming messages for later sequential dispatch
 shuffleLines = True # shuffle the order of rows/columns in each iteration while trying to send
 shuffleNeighbors = True # shuffle the order of neighbors when sending the same segment to each neighbor
 dumbRandomScheduler = False # dumb random scheduler
-segmentShuffleScheduler = True # send each segment that's worth sending once in shuffled order, then repeat
-segmentShuffleSchedulerPersist = True # Persist scheduler state between timesteps
-queueAllOnInit = False # queue up everything in the block producer, without shuffling, at the very beginning
-forwardOnReceive = True # forward segments as soon as received
-forwardWhenLineReceived = False # forward all segments when full line available (repaired segments are always forwarded)
+
+messageIsLine = False
+if messageIsLine:
+    # Message is line
+    perNodeQueue = "self.amIproposer" # keep a global queue of incoming messages for later sequential dispatch
```

Comment:
Not sure to understand this option here. Why would the block proposer need this here and the others not?

Reply:
it's not too intuitive, I know, but it is related to how we prepare the block for being pushed out from the proposer(producer): queueAllOnInit only influences the the proposer(producer). We need the perNodeQueue for queueAllOnInit to function.